### DISCUSSION TITLE
N2K PGA's

### DISCUSSION THREAD

**Discussion by huwi** [AK-Homberger/NMEA2000-Workshop#6] [Q&A]
Hallo AK!
Vielen Dank für Deine Arbeit an diesen Projekten und vor allem dass Du diese hier öffentlich machst. Ich bin gerade dabei meinen Raymarine Axiom und andere Hardware in mein Schiff einzubauen. 
Ich habe drei WIT Sensoren gekauft:
https://www.amazon.de/gp/product/B07W9LGG39/ref=ppx_yo_dt_b_asin_title_o01_s01?ie=UTF8&psc=1

Diese liefern mir unter anderem xyz-Bewegungen des Bootes. 
Mein Frage ist nun. Gibt es PGN's mit den ich die xyz-Winkel der Bootsbewegung auf dem Bus senden kann bzw. gibt es PGA's die ich selber zur freien Verfügung habe?
Schöne Grüße und Ahoi, Hubert

**Comment by AK-Homberger** [comment]
Hallo Hubert,

alle direkt durch die Bibliothek unterstützten PGNs sind in der Datei N2kMessages.h (s. Link unten) dokumentiert. Für deinen Fall würde sich PGN 127257 "Attitude" anbieten. Ob ein anderes bei dir vorhandenes Gerät, wie zum Beispiel ein Plotter, etwas damit anfangen kann bleibt offen.

https://github.com/ttlappalainen/NMEA2000/blob/1a959925d4686b5d709b654b9ddb319d3e0ca2f8/src/N2kMessages.h#L945

Viele Grüße
Andreas
